zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/ProcessInstanceRequestValidator.java

```diff
@@ -71,21 +71,25 @@ public static Optional<ProblemDetail> validateCancelProcessInstanceRequest(
   }
 
   public static Optional<ProblemDetail> validateMigrateProcessInstanceBatchOperationRequest(
-      final ProcessInstanceMigrationBatchOperationPlan request) {
+      final ProcessInstanceMigrationBatchOperationRequest request) {
```

Comment:
❓ Do we actually need to change this? Why can't we hand in the migration plan? We only use the request here to extract the migration plan now.